### ISSUE TITLE
Create a new release

### ISSUE THREAD

**Issue by sreerajkksd** [google/ngx_brotli#120]
Similar to https://github.com/google/ngx_brotli/issues/90, can we cut a new release for the latest version ?

**Comment by NicolasCARPi** [comment]
Yes, tags + releases would be great. All the arguments have been made in #90 but unfortunately there is only one RC release that was ever made.

Also note that using github actions can alleviate the manual work associated with the process.

**Comment by jirutka** [comment]
@eustas ping
